pipeline/ingestor/ingestor_common/src/test/scala/uk/ac/wellcome/platform/ingestor/common/IndexerTest.scala

```diff
@@ -0,0 +1,170 @@
+package uk.ac.wellcome.platform.ingestor.common
+
+import com.sksamuel.elastic4s.requests.analysis.Analysis
+import com.sksamuel.elastic4s.requests.mappings.dynamictemplate.DynamicMapping
+import com.sksamuel.elastic4s.{Index, Indexable}
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{FunSpec, Matchers}
+import uk.ac.wellcome.elasticsearch.IndexConfig
+import uk.ac.wellcome.elasticsearch.model.{CanonicalId, Version}
+import uk.ac.wellcome.elasticsearch.test.fixtures.ElasticsearchFixtures
+import uk.ac.wellcome.fixtures.TestWith
+import uk.ac.wellcome.json.JsonUtil._
+import uk.ac.wellcome.models.work.generators.{
+  IdentifiersGenerators,
+  WorksGenerators
+}
+import uk.ac.wellcome.platform.ingestor.common.fixtures.{
+  IngestorFixtures,
+  SampleDocument,
+  SampleDocumentData
+}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class IndexerTest
+    extends FunSpec
+    with ScalaFutures
+    with Matchers
+    with IdentifiersGenerators
+    with ElasticsearchFixtures
+    with IngestorFixtures
+    with WorksGenerators {
+
+  it("inserts an identified Work into Elasticsearch") {
```

Comment:
Assuming the test names can change to be abstract of `Work`?

Reply:
ah thanks.. I changed the test, not the title